### ISSUE TITLE
TOO MANY terminals spawn

### ISSUE THREAD

**Issue by stevemasta34** [karan/atom-terminal#31]
If you disable and re-enable the package, it will apply the keybinding +1 times and will effectively run the command that many times.

**Comment by tkw1536** [comment]
I don't think we can really fix this. Sounds like atom does not disable keybindings when disabling a package.
